lib/get-config.js

```diff
@@ -50,12 +67,124 @@ function migrateRulesFromRoot(config, options) {
   }
 }
 
-function processExtends(config) {
+function processPlugins(config, options) {
+
+  var logger = options.console || console;
+
+  var configPlugins = config.plugins;
+  var pluginsHash = {};
+
+  for (var i = 0; i < configPlugins.length; i++) {
+
+    var plugin = configPlugins[i];
+    if (typeof plugin === 'string') {
+      plugin = readPluginFromDisk(options, plugin);
+    } else if (typeof plugin !== 'object') {
+      logger.log(chalk.yellow('Plugin should be string to the plugin file or inline plugin object'));
+    }
+
+    if (plugin && plugin.name) {
+      pluginsHash[plugin.name] = plugin;
+    } else {
+      logger.log(chalk.yellow('Plugin name property is empty'));
```

Comment:
We should include the info we have in this warning/error.  I _think_ the only info we have is `configPlugins[i]` and whatever `plugin` was resolved to?